wiki/Tournaments/4DM/4/en.md

```diff
@@ -0,0 +1,354 @@
+- After each map concludes, the team with the highest team score on the map will be awarded one point. Picks will alternate between team captains until a winner is decided by being the first to get the required amount of points to win that round.
+- If team scores on a map are tied, no points will be awarded and the teams will play the map again.
+- In the Round of 32 and Round of 16, teams need to win 5 maps to win a match (best of 9). In the Quarterfinals and Semifinals, teams need to win 6 maps to win a match (best of 11). In the Finals and Grand Finals, teams need to win 7 maps to win the match (best of 13).
+- If both teams tie at match point, the tiebreaker map will be played, the team who wins the tiebreaker map will win the match.
+
+### Additional Rules
+
+The tournament hosts are responsible for scheduling all matches and informing players of when they are scheduled to play. If a team is unhappy with when they are scheduled by the hosts to play, they have the option to reschedule their match as late as the end of Thursday of the week before the weekend of their match. Instructions on how to do so will be provided in the discord server.
+
+Please note that while the organizers will try to schedule all matches for the best possible time for both teams, timezone conflicts may make some matches be forced to occur in the early mornings or late nights.
+
+- If no referee is available to referee a match, the match shall be delayed.
+- If no streamers are available to broadcast the match, the match will still happen without being broadcasted.
+- If no commentators are available to commentate the match but broadcasters are available to broadcast the match, the match will still be broadcasted but without commentary.
+
+Please note that while 4DM4 will try to broadcast every match, certain matches may be unable to be broadcasted due to overlapping matches and/or lack of streamer availability. Rescheduled matches will be prioritized under non-rescheduled matches in terms of being broadcasted.
+
+Each respective stage of the tournament is to be completed in a single weekend, using the same map pool for the entire weekend. Weekday matches are allowed if both team captains agree to them and they take place before the weekend of that scheduled stage. As aforementioned, map pools will be released roughly a week before the stage they are to be played on. The latest a map in a released map pool may be slightly updated (for example, as to fixing a notable timing or snapping issue) is by 23:59UTC of the Monday after the map pool is released.
+
+Tournament hosts have the right to revoke any signed player’s eligibility to participate in the tournament due to an obvious excess of skill compared to the tournament’s target demographic, or due to blacklist-worthy community offenses that may not see them screened out of the tournament in the Screening Phase.
```

Comment:
Note that the rules for [official tournament support](https://osu.ppy.sh/wiki/en/Tournaments/Official_support#tournaments) (this tournament is badged as it seems) require "ample evidence" and approval from the account support team in order to reject registrants that pass screening, though I suppose the next paragraph covers this

```suggestion
Tournament hosts have the right to revoke any signed player's eligibility to participate in the tournament due to an obvious excess of skill compared to the tournament's target demographic, or due to blacklist-worthy community offenses that may not see them screened out of the tournament in the screening phase.
```